### ISSUE TITLE
Top buttons issue with the replay section

### ISSUE THREAD

**Issue by KJeff01** [smogon/pokemon-showdown-client#1922]
Found a little issue at https://replay.pokemonshowdown.com/

Most of the button links at the top of the page have a strange issue only in the replay section from what I can see. There is a '?' followed by a decimal sequence that is appended after the link. The one button that doesn't exhibit this issue is the `Play` button. Just one button truly fails to work and that is the `Forums` button which brings you to a non-existent page.

![buttons](https://user-images.githubusercontent.com/22485442/149578646-cc5cfa58-1099-413e-bd00-df3696bd08f6.png)

![example](https://user-images.githubusercontent.com/22485442/149578666-368578bd-2a37-4fe4-b14d-537b9d22c350.png)

This happens on both Windows 10 and Ubuntu MATE and tested on the Google Chrome and Pale Moon browsers. Maybe this isn't the right place for this however it is something I wanted to bring attention to.

**Comment by monsanto** [comment]
Yes I can reproduce this. Strange error, I wonder what that query string is supposed to be?

**Comment by joebangara43903** [comment]
![image](https://user-images.githubusercontent.com/131228993/232965429-3e4edd01-fb90-4cf4-955c-72f8f414a3dc.png)

**Comment by Zarel** [comment]
I believe it's a Google Analytics tracker. Can either of you reproduce this on the current version?

Most of PS is designed to automatically strip them, and I did fully rewrite Replays since this report. I'm going to close this on the assumption that it's fixed, but feel free to reopen if it's still an issue.
